Felienne/hedy
# [BUG FIX] Prevents Edge from auto translating page

**Description**
In this PR we fix an issue where Edge would automatically translate English keywords to Dutch in some cases. This is undesirable as we want to decide in what language the keywords are shown and not our browser. We prevent this by manually adding the `class="notranslate"` to our layout template, which should work.

**Fixes**
This PR fixes #3208.

**How to test**
Make sure you are using Edge on a non-english language (for example Dutch). Verify that the browser no longer translates the keywords when they are set in English and therefore should be shown in English.
